Exiled.API/Features/Map.cs

```diff
@@ -377,21 +375,18 @@ public static void ExplodeEffect(Vector3 position, ProjectileType projectileType
         /// </summary>
         internal static void ClearCache()
         {
-            Room.RoomIdentifierToRoom.Clear();
-            Door.DoorVariantToDoor.Clear();
-            Lift.ElevatorChamberToLift.Clear();
-            Camera.Camera079ToCamera.Clear();
-            Window.BreakableWindowToWindow.Clear();
-            TeslaGate.BaseTeslaGateToTeslaGate.Clear();
-            Pickup.BaseToPickup.Clear();
             Item.BaseToItem.Clear();
-            TeleportsValue.Clear();
-            LockersValue.Clear();
+
+            LockersValue.RemoveAll(locker => locker == null);
```

Comment:
i can get than this work but it's not possible to do it with a Patch so ?

Reply:
i dont remember properly(i wrote that some months ago), i only remember that I couldn't do it with the patch.